content/en/containers/kubernetes/distributions.md

```diff
@@ -483,6 +484,75 @@ spec:
 More `values.yaml` examples can be found in the [Helm chart repository][1]
 More `DatadogAgent` examples can be found in the [Datadog Operator repository][2]
 
+## vSphere Tanzu Kubernetes Grid (TKG) {#TKG}
+
+TKG requires some small configuration changes, shown below. For example, tolerations are required to schedule the Node Agent on the `master` nodes.
+
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+Custom `values.yaml`:
+
+```yaml
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  kubelet:
+    tlsVerify: false
+  # Enable the new `kubernetes_state_core` check.
+  kubeStateMetricsCore:
+    enabled: true
```

Comment:
When enabling the `kubernetes_state_core` check is is better to disable the `kubernetes_state`. 

```suggestion
  # Disable the `kube-state-metrics` dependency chart installation.
  kubeStateMetricsEnabled: false
  # Enable the new `kubernetes_state_core` check.
  kubeStateMetricsCore:
    enabled: true
```

FYI. it is not needed in the operator, because it is disabled automatically when `kubernetes_state_core` is enabled. But in the helm chart, to keep backward compatibility, the user need to set 2 booleans :(